docs/source/getting_started/02_configure_pkdk.md

```diff
@@ -45,6 +45,6 @@
   ```
 - Once PeekingDuck has finished running, the processed files will be saved to the specified output directory. Open the processed file and you should get this:
 
-  <img src="images/readme/ducks_demo.gif" width="50%">
+  <img src="../../../images/readme/ducks_demo.gif" width="50%">
```

Comment:
use hardlink to the gif in github. See L28

Reply:
Updated